package.json

```diff
@@ -59,7 +59,7 @@
     "@bbc/psammead-amp-geo": "^1.1.2",
     "@bbc/psammead-assets": "^2.5.0",
     "@bbc/psammead-brand": "^5.0.5",
-    "@bbc/psammead-calendars": "^1.0.0",
+    "@bbc/psammead-calendars": "^2.0.0",
```

Comment:
This is a major version change. Are we happy to bring this in and has it been tested in Simorgh yet?

Reply:
I think so from as [this pr](https://github.com/bbc/simorgh/pull/4254) was merged yesterday:
https://github.com/bbc/simorgh/blob/e28f15b351630b90562ed85bf34d9478d02253e5/package.json#L62

Reply:
Can confirm was tested https://github.com/bbc/simorgh/pull/4254#issuecomment-542736211